server/ServerFramework.cpp

```diff
@@ -183,8 +183,9 @@ auto PythonServer::FillListPlayers(std::vector<PlayerSetupData>& players) const
     const py::extract<py::list> py_players(r);
     if (py_players.check()) {
         py::stl_input_iterator<PlayerSetupData> players_begin(py_players), players_end;
-        players.reserve(std::distance(players_begin, players_end));
-        players.insert(players.end(), players_begin, players_end);
+        players.reserve(py::len(py_players));
+        for (auto it = players_begin; it != players_end; ++it)
```

Comment:
if the `std::distance` was the problem, the leave the `.insert` ?

Reply:
Yes, restored `.insert`.